website/docs/cloud-docs/api-docs/assessment-results.mdx

```diff
@@ -0,0 +1,119 @@
+---
+page_title: Assessments - API Docs - Terraform Cloud
+description: >-
+  Assessment results contain information about continuous validation in Terraform Cloud, like drift detection. 
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+# Assessment Results API
+
+An Assessment Result is the record of an instance of continuous validation, like [Drift Detection](/cloud-docs/workspaces/settings/drift-detection).
```

Comment:
We don't capitalize feature names (I know it's weird, but it is the practice we use!).

```suggestion
An Assessment Result is the record of an instance of continuous validation, like [drift detection](/cloud-docs/workspaces/settings/drift-detection).
```